docs-js/ai-core/ai-api.mdx

```diff
@@ -0,0 +1,157 @@
+---
+id: ai-api
+title: AI API
+hide_title: false
+hide_table_of_contents: false
+description: How to use SAP Cloud SDK for AI to interact with SAP AI Core.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+  - ai core
+  - ai-api
+---
+
+The `@sap-ai-sdk/ai-api` package provides tools to manage scenarios and workflows in SAP AI Core.
+
+    - Streamline data preprocessing and model training pipelines.
+    - Execute batch inference jobs.
+    - Deploy inference endpoints for trained models.
+    - Register custom Docker registries, sync AI content from Git repositories, and register object storage for training data and model artifacts.
+
+We maintain a list of [currently available and tested AI Core APIs](https://github.com/SAP/ai-sdk-js/blob/main/docs/list-tested-APIs.md).
+
+## Installation
+
+```
+$ npm install @sap-ai-sdk/ai-api
+```
+
+:::important
+This package contains generated code.
+Updates to this package may include breaking changes.
+
+To ensure compatibility and manage updates effectively, we strongly recommend using the tilde (`~`) version range in your project instead of the caret (`^`).
+This approach will allow patch-level updates while preventing potentially breaking minor version changes.
+:::
+
+## Usage
+
+The examples below demonstrate the usage of the most commonly used APIs in SAP AI Core.
+In addition to the examples below, you can find more **sample code** [here](https://github.com/SAP/ai-sdk-js/blob/main/sample-code/src/ai-api).
+
+### Create an Artifact
+
+```ts
+async function createArtifact() {
+  const requestBody: ArtifactPostData = {
+    name: 'training-test-dataset',
+    kind: 'dataset',
+    url: 'https://ai.example.com',
+    scenarioId: 'foundation-models'
+  };
+
+  const responseData: ArtifactCreationResponse =
+    await ArtifactApi.artifactCreate(requestBody, {
+      'AI-Resource-Group': 'default'
+    }).execute();
+  return responseData;
+}
+```
+
+### Create a Configuration
+
+```ts
+async function createConfiguration() {
+  const requestBody: ConfigurationBaseData = {
+    name: 'gpt-35-turbo',
```

Comment:
```suggestion
    name: 'gpt-4o',
```

We should create a bot, which regularly scans docs and updates deprecated models with it's replacement.

Reply:
I was using an old version of the readme. Thanks for pointing them out.